.travis.yml

```diff
@@ -33,26 +34,41 @@ jobs:
     - stage: "dep check"
       script: make depcheck
       name: " Check if imports, Gopkg.toml, and Gopkg.lock are in sync"
-    - stage: "Test Jobs"                # naming the Tests stage
+
+    - stage: "Test Build"                # naming the Tests stage
       script: make 
       name: "builds cloud and edge components"            # names the first Tests stage job
+    - script: make edge_cross_build
+      name: "cross build edge"     # names the fifth Tests stage job
+    - script: make bluetoothdevice
+      name: "build bluetooth_device"
+    - script: make edge_small_build
+      name: "build small edge version"
+    - stage: "Test lint"
```

Comment:
lets move the lint stage before the build stage ?

Reply:
You are right, lint fails more often than the build in the travis tests